### ISSUE TITLE
How to login in Webui

### ISSUE THREAD

**Issue by algorys** [Alignak-monitoring-contrib/alignak-webui#4]
Hello,

I've installed `Alignak 0.2` and `alignak-webui 0.2.3`. All seems to working very well. I've reach the webui but I can't login inside ...

I've search in your docs, but can't find something to similar to Shinken... I've a file contact admin.cfg with password default (admin / admin) but it seems not work.

Does I have to precise method of authentication somewhere in alignak-webui ?

Otherwise great work.

Thanks for help.

**Comment by mohierf** [comment]
This version is still a development version ... and I did not updated since a long time :/

The user authentication is made with a user in Alignak backend, so you need to install Alignak backend module (https://github.com/Alignak-monitoring-contrib/alignak-backend) and then feed some data in the backend.

 The process is not yet very easy ... and not well documented ...

 further more, I am preparing another version of this Web UI which aims to replace the current version stored in the repo

**Comment by algorys** [comment]
> This version is still a development version ... and I did not updated since a long time :/

Yes I know, but I'm interested by this project (all Alignak's in general), so I want to try it before maybe help.

Ok I will try to install backend, I'm not sure if it was necessary or not.

> The process is not yet very easy ... and not well documented ...

I've see that ;) But by searching the Internet , I managed to install a base

> further more, I am preparing another version of this Web UI which aims to replace the current version stored in the repo

This version is based on the webui2 of Shinken, isn't it ?

Is there a repo for your next version ?

Thanks

**Comment by mohierf** [comment]
My new version has not yet a repo ... I think I will post a beta in a few days ... probably next week :-)

For any help on alignak backend, irc channel alignak is your best friend ;-)

**Comment by algorys** [comment]
Ok, so I'll waiting for that I think.
